test/Npgsql.Tests/Types/ArrayTests.cs

```diff
@@ -595,6 +363,26 @@ public async Task Read_two_empty_arrays()
         Assert.AreNotSame(reader.GetFieldValue<List<int>>(0), reader.GetFieldValue<List<int>>(1));
     }
 
+    [Test]
+    public async Task Arrays_not_supported_by_default_on_NpgsqlSlimSourceBuilder()
+    {
+        var dataSourceBuilder = new NpgsqlSlimDataSourceBuilder(ConnectionString);
+        await using var dataSource = dataSourceBuilder.Build();
+
+        await AssertTypeUnsupportedRead<int[], InvalidCastException>("{1,2,3}", "integer[]", dataSource);
+        await AssertTypeUnsupportedWrite<int[], InvalidCastException>(new[] { 1, 2, 3 }, "integer[]", dataSource);
+    }
+
+    [Test]
+    public async Task NpgsqlSlimSourceBuilder_EnableRanges()
```

Comment:
EnableRanges?

Reply:
You mean as the test name, instead of the full NpgsqlSlimSourceBuilder_EnableRanges name? For now that's the pattern I've done across the other types (e.g. NpgsqlSlimSourceBuilder_EnableFullTextSearch). Is seems like good naming to me, but if you care I don't mind changing across the board.